includes/Core/Modules/Modules.php

```diff
@@ -567,9 +567,24 @@ protected function get_registry() {
 	 */
 	protected function setup_registry() {
 		$registry = new Module_Registry();
-
-		foreach ( $this->core_modules as $core_module ) {
-			$registry->register( $core_module );
+		/**
+		 * Filters core module slugs before registering them in the module registry. Each slug presented on this array will
+		 * be registered for inclusion. If a module is forced to be active, then it will be included even if the module slug is
+		 * removed from this filter.
+		 *
+		 * @param array $available_modules An array of core module slugs available for registration in the module registry.
+		 *
+		 * @since n.e.x.t
+		 *
```

Comment:
A bit of a nit-pick, but `@param` should always come right before `@return` (without blank line), and `@since` should be before `@param` (with blank line in between).